query/outputnode_test.go

```diff
@@ -262,3 +263,43 @@ func TestChildrenOrder(t *testing.T) {
 	}
 	require.Nil(t, child)
 }
+
+func TestMarshalTimeJson(t *testing.T) {
+	var timesToMarshal = []struct {
+		in  time.Time
+		out string
+		err string
+	}{
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", -6*60*60)),
+			out: "\"2018-05-30T09:30:10-06:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 500000000, time.UTC),
+			out: "\"2018-05-30T09:30:10.5Z\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", -23*60*60)),
+			out: "\"2018-05-30T09:30:10-23:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", -24*60*60)),
+			out: "\"2018-05-30T09:30:10-24:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", 23*60*60)),
+			out: "\"2018-05-30T09:30:10+23:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", 24*60*60)),
+			out: "\"2018-05-30T09:30:10+24:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", -30*60*60)),
+			out: "\"2018-05-30T09:30:10-30:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", 30*60*60)),
+			out: "\"2018-05-30T09:30:10+30:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", -99*60*60)),
+			out: "\"2018-05-30T09:30:10-99:00\""},
+		{in: time.Date(2018, 5, 30, 9, 30, 10, 0, time.FixedZone("", 99*60*60)),
+			out: "\"2018-05-30T09:30:10+99:00\""},
+	}
+
+	for _, tc := range timesToMarshal {
+		out, err := marshalTimeJson(tc.in)
+		if tc.err == "" {
```

Comment:
`tc.err` will always be "", ya? did you mean to add some failing fixtures in the timesToMarshall slice?

Reply:
Done